go/indexer/metrics/metrics.go

```diff
@@ -43,7 +43,7 @@ func NewMetrics(monitoredTokens map[string]string) *Metrics {
 
 	return &Metrics{
 		SyncHeight: promauto.NewGaugeVec(prometheus.GaugeOpts{
-			Name:      "l1_sync_height",
+			Name:      "sync_height",
```

Comment:
I see we are using a single metric with l1/l2 labels. Should the `Help` comment below be updated to reflect the same?

Reply:
Yeah good catch. Fixing now.